org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/BatchCompilerTest.java

```diff
@@ -10273,28 +10247,6 @@ public void test285_option_files() {
         true /*shouldFlushOutput*/);
 }
 
-// command-line expansion
-public void test286_option_files() {
-	runNegativeTest(
-		new String[] {
-			"X.java",
-			"public @interface X {\n" +
-			"}",
-			"options.txt",
-			"-source 1.4"
-		},
-        "\"" + OUTPUT_DIR +  File.separator + "X.java\" " +
-        "\"@" + OUTPUT_DIR +  File.separator + "options.txt\"",
-        "" /* expectedOutOutputString */,
-        "----------\n" + /* stderr */
-        "1. ERROR in ---OUTPUT_DIR_PLACEHOLDER---/X.java (at line 1)\n" +
-        "	public @interface X {\n" +
-        "	                  ^\n" +
-        "Syntax error, annotation declarations are only available if source level is 1.5 or greater\n" +
-        "----------\n" +
-        "1 problem (1 error)\n",
-        true /*shouldFlushOutput*/);
-}
 // command-line expansion
 // shows that we don't recurse
 public void test287_option_files() {
```

Comment:
Could not readily understand this change

Reply:
Test removed as it tested compilation behavior of 1.4 JLS. There is no error anymore, as we compile with 1.8 JLS minimal.